hazelcast/src/test/java/com/hazelcast/map/impl/operation/QueryPartitionOperationTest.java

```diff
@@ -0,0 +1,52 @@
+package com.hazelcast.map.impl.operation;
+
+import com.hazelcast.map.QueryResultSizeExceededException;
+import com.hazelcast.map.impl.QueryResult;
+import com.hazelcast.query.TruePredicate;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class QueryPartitionOperationTest extends QueryOperationTestSupport {
+
+    @Override
+    protected AbstractMapOperation createQueryOperation() {
+        return new QueryPartitionOperation(MAP_NAME, TruePredicate.INSTANCE);
+    }
+
+    @Test
+    public void testConstructor() {
+        new QueryPartitionOperation();
```

Comment:
I see that very differently:
* It provided the only usage in code for the default constructor, so there is no need for a `@SuppressWarnings("unused")` in the class.
* It ensures that the default constructor doesn't get deleted, because it looks unused, which will cause runtime exceptions, not compile time exceptions.
* It increases code coverage with a unit test (not all tests are executed in all builds, especially the slow nightly integration tests which will use the constructor).